zap/src/test/java/org/zaproxy/zap/extension/pscan/scanner/DefaultRegexAutoTagScannerTest.java

```diff
@@ -115,22 +118,80 @@ private RegexAutoTagScanner getRegexRuleByName(String taggerName) {
                 "aPPlication/json",
                 "application/json; charset=utf-8"
             })
-    void shouldCountWhenHeaderContainsJsonMatch(String contentType)
-            throws URIException, HttpMalformedHeaderException, DatabaseException {
+    void shouldCoutWhenHeaderMatchesJsonTag(String contentType) {
```

Comment:
There are some typos being introduced in the method names (Count).

Reply:
DOH, thank you